src/Utils/SchemaPrinter.php

```diff
@@ -337,16 +341,22 @@ protected static function printObject(ObjectType $type, array $options): string
      */
     protected static function printFields(array $options, $type): string
     {
-        $fields = [];
+        $items = [];
```

Comment:
Let's keep this as `$fields` and change the other variable to `$fieldDefinitions`. Same with the other changes like this.

Reply:
@spawnia Done!